Gemfile

```diff
@@ -11,7 +11,7 @@ if RUBY_VERSION >= "2.3.0"
     gem 'guard-rspec'
   end
 else
-  gem "rails", ">= 3.0", "< 5.0"
+  gem "rails", ">= 3.0", "<= 5.1"
```

Comment:
> Travis CI builds
Rails 5.1 already specifies `required_ruby_version` >= 2.2.2, so this is not necessary.

> and developers 
This will not affect the developers because the dependency spec is evaluated when building the gem, not when installing it.